notebooks/output_randomized_smoothing_mnist.ipynb

```diff
@@ -109,128 +112,115 @@
-    "                             use_logits=False)"
+    "def train_step(model, images, labels):        \n",
+    "    with tf.GradientTape() as tape:\n",
+    "        predictions = model(images, training=True)\n",
+    "        loss = loss_object(labels, predictions)\n",
+    "    gradients = tape.gradient(loss, model.trainable_variables)\n",
+    "    optimizer.apply_gradients(zip(gradients, model.trainable_variables))\n",
+    "    \n",
+    "loss_object = tf.keras.losses.CategoricalCrossentropy(from_logits=False)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {
-    "colab": {
-     "base_uri": "https://localhost:8080/",
-     "height": 221
-    },
+    "colab": {},
     "colab_type": "code",
-    "id": "qH0lH14Ddv4d",
-    "outputId": "058fe69b-70af-46ad-8d71-bc7f923f2202"
+    "id": "tGbe8Cjmdv4a"
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "WARNING:tensorflow:From /Users/sharon/anaconda/envs/tensorflow/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3445: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.\n",
-      "Instructions for updating:\n",
-      "Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.\n",
-      "Epoch 1/3\n",
-      "469/468 [==============================] - 30s 63ms/step - loss: 0.2744 - acc: 0.9172\n",
-      "Epoch 2/3\n",
-      "469/468 [==============================] - 30s 63ms/step - loss: 0.0821 - acc: 0.9745\n",
-      "Epoch 3/3\n",
-      "469/468 [==============================] - 30s 64ms/step - loss: 0.0609 - acc: 0.9807\n",
-      "Epoch 1/3\n",
-      "469/468 [==============================] - 31s 66ms/step - loss: 0.3985 - acc: 0.8738\n",
-      "Epoch 2/3\n",
-      "469/468 [==============================] - 32s 69ms/step - loss: 0.1520 - acc: 0.9509\n",
-      "Epoch 3/3\n",
-      "469/468 [==============================] - 32s 67ms/step - loss: 0.1150 - acc: 0.9633\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "# add Gaussian noise and train two classifiers\n",
-    "sigma1 = 0.25\n",
-    "sigma2 = 0.5\n",
+    "# Construct and train a convolutional neural network in standard (non-smoothed) classifier\n",
     "\n",
-    "# sigma = 0.25\n",
-    "ga = GaussianAugmentation(sigma=sigma1, augmentation=False)\n",
-    "x_new1, _ = ga(x_train)\n",
+    "classifier = TensorFlowV2Classifier(model=get_model(input_shape, min_, max_),\n",
+    "                                    nb_classes=nb_classes,\n",
+    "                                    input_shape=input_shape,\n",
+    "                                    loss_object=loss_object,\n",
+    "                                    train_step=train_step,\n",
+    "                                    channel_index=3,\n",
```

Comment:
Replace `channel_index` in favor of `channels_first`.

Reply:
Ok.